### PR TITLE
[4.0] Table Column alignment (part3) [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
Remaining tables - Continuation of #21933

_nowrap will be addressed in its own pr_

### CODE REVIEW

File: administrator/components/com_plugins/tmpl/plugins/default.php [hunk 1]
```diff
@@ -113,16 +113,16 @@
...
-						<td class="nowrap small d-none d-md-table-cell text-center">
+						<td class="nowrap small d-none d-md-table-cell">
 							<?php echo $this->escape($item->folder); ?>
 						</td>
-						<td class="nowrap small d-none d-md-table-cell text-center">
+						<td class="nowrap small d-none d-md-table-cell">
 							<?php echo $this->escape($item->element); ?>
 						</td>
-						<td class="small d-none d-md-table-cell text-center">
+						<td class="small d-none d-md-table-cell ">
```
Comment (Quy) [inline administrator/components/com_plugins/tmpl/plugins/default.php]: Remove space

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on 4231d6ae320061a713f649e92a0d562ea67731c2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21967">issues.joomla.org/tracker/joomla-cms/21967</a>.</sub>

**Comment by zwiastunsw** [comment]
I have tested this item :white_check_mark: successfully on 4231d6ae320061a713f649e92a0d562ea67731c2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21967">issues.joomla.org/tracker/joomla-cms/21967</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
Ready to Commit after two successful tests.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**